eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -24,9 +24,9 @@
 # 17. Produce plots DONE
 """
 
-import sys, os, time
+import os, time
 import numpy as np
-import multiprocessing as mp
```

Comment:
Why remove multiprocessing as an import?

Reply:
It's currently not used in s3_reduce.py. I moved the background initialization into lib/util.py a while back